frontend/lib/screens/new_student/local_widget/skills_form.dart

```diff
@@ -66,18 +66,22 @@ class SkillsForm extends StatelessWidget {
       "softSkills": skillsBoxes[1].chosenSkills
     };
 
-    Student newStudent = await APIService.route(
+    dynamic response = await APIService.route(
       ENDPOINTS.Student,
       "/student",
       body: student,
     );
 
-    if (newStudent != null) {
-      dynamic response = await APIService.route(
-          ENDPOINTS.Student, "/student/id/skills",
-          urlArgs: newStudent.id, body: student.skills);
-      print(response.toString());
+    String message;
+    if (response is Student) {
+      message = "Profile saved successfully";
+      Nav.navigatorKey.currentState.popAndPushNamed("/home");
+    } else if (response is String) {
+      message = response;
```

Comment:
This error displays correctly, but the student is created anyway on the DB without education and work experience